ansible/playbook.yml

```diff
@@ -179,10 +179,10 @@
   - role: gcsfuse
     when:
     - install_gcsfuse
-    - ansible_architecture == "x86_64"
+    - ansible_architecture == "x86_64" or (ansible_distribution == "Ubuntu" and ansible_distribution_version is version('24.04', '=='))
```

Comment:
I think you are trying to say "x86 only unless it's Ubuntu 24.04" but I read the GCS Fuse overview as saying ARM64 is pretty widely supported now:

https://cloud.google.com/storage/docs/cloud-storage-fuse/overview

GitHub releases suggest this was added in Sept. 2023. I would probably simply remove the 2nd condition entirely.

Reply:
Removed architecture check